standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -220,6 +224,8 @@ protected Connection getConnection() {
       rs = fetchColumnsViaDBMetaData(tableName);
       List<FieldSchema> cols = new ArrayList<>();
       while (rs.next()) {
+        String datatype = rs.getString("DATA_TYPE");
```

Comment:
these 2 do not appear to be in use. Was it your intent to use one of them in the setType()

Reply:
Need to  remove this unused variable